internal/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
+add_subdirectory(nlohmannJson)
+
+set(FABULIST_THIRDPARTY_JSON OFF)
```

Comment:
This should ideally be set inside of Fabulist's dependency CMakeLists, before the `MakeAvailable` call